Formula/tflint.rb

```diff
@@ -0,0 +1,25 @@
+class Tflint < Formula
+  desc "Terraform linter that look for errors not detected by `terraform plan`"
+  homepage "https://github.com/wata727/tflint"
+  url "https://github.com/wata727/tflint/archive/v0.5.3.tar.gz"
+  sha256 "a46b32c4f67aa75d03e1b1a1f63d6134bdec87467f93cb551109a5db4a3f9429"
+
+  depends_on "glide" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home"
+    (buildpath/"src/github.com/wata727/tflint").install buildpath.children
+    cd "src/github.com/wata727/tflint" do
+      system "glide", "install"
+      system "go", "build", "-o", "tflint"
+
+      bin.install "tflint"
+    end
+  end
+
+  test do
+    system "#{bin}/tflint", "--version"
```

Comment:
We'll need a test that actually does check the functionality of the software. Running it against a file with expected error, for example?